editor/src/components/editor/import-wizard/components.tsx

```diff
@@ -0,0 +1,320 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx } from '@emotion/react'
+import React from 'react'
+import type {
+  ImportCheckRequirementAndFix,
+  ImportFetchDependency,
+  ImportOperation,
+} from '../../../core/shared/import/import-operation-types'
+import { ImportOperationResult } from '../../../core/shared/import/import-operation-types'
+import { assertNever } from '../../../core/shared/utils'
+import { Icons } from '../../../uuiui'
+import { GithubSpinner } from '../../../components/navigator/left-pane/github-pane/github-spinner'
+import { RequirementResolutionResult } from '../../../core/shared/import/proejct-health-check/utopia-requirements-types'
+
+export function OperationLine({ operation }: { operation: ImportOperation }) {
+  const operationRunningStatus = React.useMemo(() => {
```

Comment:
the `useMemo` is not necessary here, because the return value is a string, which has value equality semantics by default

Reply:
I was actually trying to save the internal comparisons from running every render if they don't need to (similar to what a computed field would have done)